src/test/java/org/apache/log4j/README.md

```diff
@@ -0,0 +1,21 @@
+# Reason for these files
+
+Spark uses very old Log4j dependency. We don't want to include it since it is not maintained and full of vulnerabilities.
```

Comment:
Can't we override the dependency with a newer one?

Reply:
Please configure branch protection (using github-keeper)

Reply:
I could not unfortunately after some trying out. The error is from latest version of log4j. 

Mainly the method mentioned in the error is not available in latest versions.